index.bs

```diff
@@ -872,88 +872,93 @@ When this method is invoked, the user agent MUST execute the following algorithm
     [=AbortSignal/aborted flag=] is set to [TRUE], return a {{DOMException}} whose name is "{{AbortError}}"
     and terminate this algorithm.
 
-1. Start |lifetimeTimer|.
-
 1. Let |issuedRequests| be a new [=ordered set=].
 
-1. [=set/For each=] |authenticator| that becomes available on this platform during the lifetime of |lifetimeTimer|, do the
-    following:
+1. Let |authenticators| represent a [=set=] of platform-specific handles, where each value identifies an [=authenticator=]
```

Comment:
The reason I chose the word "represent" here is to hint at how this differs from a "normal" set in that it's an abstract reference to something that may change over time, as opposed to a normal set which only changes when methods are called on it.

Come to think of it, the only purpose of |authenticators| is to be able to then say "[=set/for each=] |authenticator| in |authenticators|". It's not used for anything else, so perhaps it might be better to try to merge its definition into the "for each authenticator" step. That would make the latter pretty unwieldy, though.

Reply:
That said, I like your suggestion, but I think it's necessary to call out the non-constant nature of this value. How about this?

18. Let |authenticators| represent a value which at any given instant is a [=set=] of platform-specific handles, where each
    [=set/item=] identifies an [=authenticator=] presently available on this platform at that instant.

Reply:
I like the above re-phrased step 18, it seems clearer wrt what's intended, thx.